README.md

```
@@ -1 +1,346 @@
+## 📢 폴더 구조
+
+1. **`shared/`** 아래 폴더는 전부 common(공통)의 의미로 생각한다.
+2. **`pages/`** 아래 세부 폴더(components, constants 등등)가 각각 위치한다.
+
+```
+|-- 📁 .github
+|-- 📁 node_modules
+|-- 📁 public
+|-- 📁 src 
+  |-- 📁 pages
+	  |-- 📁 home
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)